tools/topology/sof/pipe-smart-amplifier-playback.m4

```diff
@@ -78,7 +88,8 @@ C_CONTROLBYTES(Smart_amp Model, PIPELINE_ID,
 W_PCM_PLAYBACK(PCM_ID, Smart Amplifier Playback, 2, 0)
 
 # Mux 0 has 2 sink and source periods.
-W_SMART_AMP(0, PIPELINE_FORMAT, 2, 2, LIST(`             ', "Smart_amp Config", "Smart_amp Model"))
+ifelse(SOF_ABI_VERSION_3_17_OR_GRT, `1',
+W_SMART_AMP(0, PIPELINE_FORMAT, 2, 2, LIST(`             ', "Smart_amp Config", "Smart_amp Model", "Smart_amp Model_Get_params")))
```

Comment:
@RDharageswari you need to change this to add the first 2 controls by default and the get_params control only if the abj is >= 3.17

Reply:
Done